src/transformers/models/maskformer/configuration_maskformer.py

```diff
@@ -211,6 +211,5 @@ def to_dict(self) -> Dict[str, any]:
         """
         output = copy.deepcopy(self.__dict__)
         output["backbone_config"] = self.backbone_config.to_dict()
-        output["decoder_config"] = self.decoder_config.to_dict()
```

Comment:
Any reason this is deleted?

Reply:
This was copied from configuration_maskformer.py but the test_config test is failing, I'm fixing it

Reply:
I'm not sure this should be changed for MaskFormer